### ISSUE TITLE
Cannot get an existing Environment

### ISSUE THREAD

**Issue by desteves** [pulumi/pulumi-pulumiservice#319]
### What happened?

I'm attempting to get an existing Environment using the `pulumiservice.Environment` resource but I get `error: Preview failed: failed to unmarshal organization value from properties: map[]`

### Example

`index.ts`

```typescript
import * as pulumi from "@pulumi/pulumi";
import * as pulumiservice from "@pulumi/pulumiservice";


// pulumi config set envName aws-oidc-env

const config = new pulumi.Config();
const org = pulumi.getOrganization();
const envName = config.require("envName");
const envId =  `${org}/${envName}`;

const envRef =  pulumiservice.Environment.get(envId, envId);
```

### Output of `pulumi about`

CLI          
Version      3.119.0
Go Version   go1.22.3
Go Compiler  gc

Plugins
KIND      NAME           VERSION
language  nodejs         unknown
resource  pulumiservice  0.21.2

Host     
OS       darwin
Version  14.4.1
Arch     arm64

This project is written in nodejs: executable='/opt/homebrew/bin/node' version='v22.2.0'

### Contributing

Vote on this issue by adding a 👍 reaction. 
To contribute a fix for this issue, leave a comment (and link to your pull request, if you've opened one already).

**Comment by IaroslavTitov** [comment]
I have reproduced the issue, but it so happened to be solved in https://github.com/pulumi/pulumi-pulumiservice/pull/326, which I was working on today. I will keep the issue open until I cut a new release, after that updating to new version will fix the issue.

**Comment by IaroslavTitov** [comment]
The new release containing a fix is out - 0.21.4, resolving
